jomar-restrepo/tarea2/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html semantico</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=League+Gothic&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main>
+      <section class="fondo">
+        <section class="titulo">
+          <section class="subtitulo">
+            <img
+              class="logo"
+              src="/jomar-restrepo/tarea2/img/logotipo-de-playstation.png"
```

Comment:
@Leviathan9020 , you should use realtive paths and <figure> tag

```suggestion
              src="img/logotipo-de-playstation.png"
```